test/mocks/examples/gcp-pubsub-publisher.architect.yml

```diff
@@ -16,7 +16,7 @@ dependencies:
 services:
```

Comment:
The only two files that are used here anymore are `hello-world` and `database-seeding`. None of the others need to be kept around.

Reply:
removed extra projects that were not being utilized in tests.